multitask_exp.py

```diff
@@ -0,0 +1,90 @@
+import os
+import tensorflow as tf
+from lib import multitask as mt
+from utils import args_util, training
+
+def main():
```

Comment:
Not necessarily, but I think this function can be broken into a few smaller understandable stages / functions.